### ISSUE TITLE
require_partition_filter is deprecated in .terraform/modules/bigquery/main.tf line 67, in resource "google_bigquery_table" "main"

### ISSUE THREAD

**Issue by habur6** [terraform-google-modules/terraform-google-bigquery#311]
### TL;DR

require_partition_filter in time_partitioning block is deprecated. This is causing the warning below while using the module.
require_partition_filter in time_partitioning is still mandatory to use the module. Please upgrade to the latest version of https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_table

As per below url, the documentation mentiones the deprecation.
[require_partition_filter](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_table#require_partition_filter) - (Optional) If set to true, queries over this table require a partition filter that can be used for partition elimination to be specified. require_partition_filter is deprecated and will be removed in a future major release. Use the top level field with the same name instead.

│ Warning: Argument is deprecated
│ 
│   with module.bigquery.google_bigquery_table.main["recsrc_gtm_events"],
│   on .terraform/modules/bigquery/main.tf line 67, in resource "google_bigquery_table" "main":
│   67: resource "google_bigquery_table" "main" {
│ 
│ This field is deprecated and will be removed in a future major release;
│ please use the top level field with the same name instead.
│ 
│ (and 2 more similar warnings elsewhere)

**Comment by rhettBIngka** [comment]
+1

**Comment by mlopezgez** [comment]
+1

**Comment by japerry911** [comment]
+1
